modules/navier_stokes/src/bcs/INSFEFluidEnergyDirichletBC.C

```diff
@@ -27,7 +27,10 @@ INSFEFluidEnergyDirichletBC::validParams()
   params.addCoupledVar("w", "velocity in z-direction"); // required in 3D
 
   params.addParam<FunctionName>("v_fn", "Velocity function with time at the boundary");
-  params.addParam<FunctionName>("T_fn", "Temperature function with time at the boundary");
+
+  params.addParam<Real>("T_scale", 1.0, "Coefficient to multiply by the temperature");
+  params.addParam<FunctionName>("T_fn", "1", "A function that describes the temperature");
```

Comment:
are you sure about providing a default?
seems very risky, a user could forget a parameter and get nonsense